web/packages/build/package.json

```diff
@@ -24,7 +24,7 @@
     "eslint-plugin-jest": "^29.0.1",
     "eslint-plugin-jest-dom": "^5.5.0",
     "eslint-plugin-react": "^7.37.5",
-    "eslint-plugin-react-hooks": "6.0.0-rc.2",
+    "eslint-plugin-react-hooks": "7.0.1",
```

Comment:
`preserve-manual-memoization` is still noisy, so I'm not re-enabling it.

I check if it's noisy by turning it into a warning, restarting my LSP and then checking if it shows a big warning throughout this function:

https://github.com/gravitational/teleport/blob/eadc66828067298fca3b4a23ed8a8349d0c92e37/web/packages/teleport/src/WorkloadIdentity/WorkloadIdentities.tsx#L84-L104

It seems that the problem is `pageToken`. React Compiler thinks it can be mutated later. I guess it's true but I don't see how it's different from other deps in this callback.

Someone made an issue about it: https://github.com/facebook/react/issues/34924